src/documentation/markdown/2021/static-dataset-faq.md

```diff
@@ -1,30 +1,30 @@
-## Static Dataset FAQ
+# Static Dataset FAQ
 
-### What is the difference between the static datasets?
+## What is the difference between the static datasets?
 The primary difference between the datasets is the time period when the data is captured. Each time period contains the most up-to-date data _at the moment in time when the dataset was created_.
 
 |Dataset|Time Period|
 |---|---|
-|Snapshot | 1 month after the timely filing deadline |
+|Snapshot | Typically 1 month |
```

Comment:
```suggestion
|Snapshot | Typically 1 month after the timely filing deadline|
```

Reply:
Change was made due to Chynna's comment on [table of content](https://github.com/cfpb/hmda-frontend/issues/1487) ticket.

> 5. [static-dataset-faq.md](https://github.com/cfpb/hmda-frontend/blob/1487-table-of-contents/src/documentation/markdown/2018/static-dataset-faq.md) -
> - Change: For 'Snapshot' row listed in the table, change "1 month after the timely filing deadline "to "Typically 1 month" - so that we are not held to April 1 every year.